opsmngr/automation_config.go

```diff
@@ -80,19 +90,25 @@ type AutomationConfig struct {
 	LDAP                 *map[string]interface{}   `json:"ldap,omitempty"`
 	MongoDBToolsVersion  *map[string]interface{}   `json:"mongoDbToolsVersion,omitempty"`
 	MongoDBVersions      []*map[string]interface{} `json:"mongoDbVersions,omitempty"`
-	MongoSQLDs           []*map[string]interface{} `json:"mongosqlds,omitempty"`
+	MongoSQLDs           []*map[string]interface{} `json:"mongosqlds"`
 	MonitoringVersions   []*ConfigVersion          `json:"monitoringVersions,omitempty"`
-	OnlineArchiveModules []*map[string]interface{} `json:"onlineArchiveModules,omitempty"`
-	MongoTS              []*map[string]interface{} `json:"mongots,omitempty"`
+	OnlineArchiveModules []*map[string]interface{} `json:"onlineArchiveModules"`
+	MongoTS              []*map[string]interface{} `json:"mongots"`
 	Options              *map[string]interface{}   `json:"options"`
 	Processes            []*Process                `json:"processes"`
 	ReplicaSets          []*ReplicaSet             `json:"replicaSets"`
 	Roles                []*map[string]interface{} `json:"roles"`
 	Sharding             []*ShardingConfig         `json:"sharding"`
 	SSL                  *SSL                      `json:"ssl,omitempty"`
 	TLS                  *SSL                      `json:"tls,omitempty"`
-	UIBaseURL            *string                   `json:"uiBaseUrl"`
+	UIBaseURL            *string                   `json:"uiBaseUrl,omitempty"`
 	Version              int                       `json:"version,omitempty"`
+	raw                  []byte
```

Comment:
[q] do we really want to keep this? I guess it served as a way to debug the fields but it would be doubling the memory space with raw data and the struct.

If you feel like this adds value and want to keep it, I think a better way would be to add the raw field to the [Response](https://github.com/mongodb/go-client-mongodb-atlas/blob/049456a13b6ed15551f7e4cfd2f30f1c2134bb21/mongodbatlas/mongodbatlas.go#L125-L130) struct we return, and maybe add a `withRaw` option to the `Client` so people can opt in into getting the raw bytes for debug purposes and we can keep a low memory usage and also support this in other methods

Reply:
We need it to be able to backup origin automation config before changing it on client application or to validate the difference between parsed config and the original one.
Yeah. I thought to add it to `Response` struct, but `atlas.Response` looks very flexible so it can or unmarshal data directly to our struct or write raw bytes. It's why I don't want to add it to `Response` struct. But I really like the option `withRaw` in client size. But in that case, I think `Raw()` method should return marshaled struct. What do you think?

Reply:
Reimplemented Raw in atlas.Response. Thanks!